docs/Makefile

```diff
@@ -26,12 +26,6 @@ STEPS = discover provision prepare execute finish report
 # A list of `plugins/*.rst` files to generate
 PLUGIN_TARGETS = $(addsuffix .rst,$(addprefix plugins/,$(STEPS))) plugins/test-checks.rst
 
-# A source of logo picture for Sphinx docs favicon
-LOGO_SRC = https://raw.githubusercontent.com/teemtee/docs/main/logo/tmt-small.png
-
-# Local filepath to fetched logo
-LOGO_DST = _static/tmt-small.png
-
```

Comment:
Seems ok to drop the logo download. Unless github refuses to serve the raw content to the regular users as well ;-) @happz, any concerns here?

Reply:
I believe the downloaded file was not used at all. It was replaced with the remote one in https://github.com/teemtee/tmt/commit/9d19728b084d184e5727ab443d222402f46e7bd9

Reply:
Ack, and if github decides to stop serving the raw files, we'll fix it. Hope they will not.